extendedmodlog/eventmixin.py

```diff
@@ -1174,17 +1172,16 @@ async def on_message_edit(self, before: discord.Message, after: discord.Message)
         fmt = "%H:%M:%S"
         if embed_links:
             embed = discord.Embed(
-                description=before.content,
+                description=f"**Message edited in {before.channel.mention}** [Jump to message]({after.jump_url})",
                 colour=await self.get_event_colour(guild, "message_edit"),
                 timestamp=before.created_at,
             )
-            jump_url = f"[Click to see new message]({after.jump_url})"
-            embed.add_field(name=_("After Message:"), value=jump_url)
-            embed.add_field(name=_("Channel:"), value=before.channel.mention)
+            embed.add_field(name=_("Before:"), value=before.content, inline=False)
+            embed.add_field(name=_("After:"), value=after.content, inline=False)
```

Comment:
`after.content` is redundant information and clutters the logs. There's no need to compare the final message in the logs itself and deleted messages are merely for reference. Furthermore embed fields are limited to 1024 characters so if anyone edits their message with more than 1024 characters this will not post at all defeating the purpose of logging. This is why *only* the jump link was previously provided.